internal/app/networkd/pkg/networkd/networkd.go

```diff
@@ -264,6 +276,10 @@ func (n *Networkd) Reset() {
 	for _, iface := range n.Interfaces {
 		iface.Reset()
 	}
+
+	// Empty out resolv.conf
+	//nolint: errcheck
+	ioutil.WriteFile("/etc/resolv.conf", []byte{}, 0644)
```

Comment:
Maybe we should log the error here?

Reply:
To be consistent with how we are doing it in other places ... something like:

```go
log.Printf("WARNING: failed to ....: %v", err)
```

Reply:
id rather address it when we get better logging in place altogether. `iface.Reset()` can throw errors as well, but we silently ignore them for now

Reply:
A simple log here wouldn't hurt, and I'm not sure what "better logging" buys us other than maybe a convenience `log.Warn` function. If this fails to write, we will banging our heads against the wall trying to figure out why `/etc/resolv.conf` isn't correct.

Reply:
I ended up dropping this since it's not necessary.